assets/chezmoi.io/docs/user-guide/machines/windows.md

```diff
@@ -28,3 +28,23 @@ if (-Not ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdent
   }
 }
 ```
+
+If you use [gsudo](https://gerardog.github.io/gsudo/docs/intro), their documentation specifies how to write self-elevating scripts [here](https://gerardog.github.io/gsudo/docs/tips/script-self-elevation#self-elevate-script).
```

Comment:
I would rewrite this to:

```markdown
If you use [gsudo](https://gerardog.github.io/gsudo/docs/intro),
it has tips on writing [self-elevating scripts](https://gerardog.github.io/gsudo/docs/tips/script-self-elevation#self-elevate-script).
```

It’s generally bad practice to put a link behind `[here]`, because the context is lost.

Reply:
This change should probably not be optional, because it’s an accessibility issue. The rest are verbosity/tone suggestions.